test/e2e/slack_test.go

```diff
@@ -116,7 +116,6 @@ func TestSlack(t *testing.T) {
 		command := "filters list"
```

Comment:
As Huseyin wrote, it would be great to cover that in E2E tests. We have two Slack channels, so we can easily test different bindings 👍